docs/fhir-guide/FHIR 101 - Practical Guide.ipynb

```diff
@@ -0,0 +1,2966 @@
+    "\n",
+    "You don't need to do this if you would like to use another HTTP client like `Postman` to interact with the FHIR server."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "hidden": true
+   },
+   "outputs": [],
+   "source": [
+    "# Be sure to execute this cell so that we can use the client later\n",
+    "from pprint import pprint\n",
+    "from kf_model_fhir.client import FhirApiClient\n",
+    "\n",
+    "BASE_URL = 'http://hapi.fhir.org/baseR4'\n",
+    "FHIR_VERSION = '4.0.0'\n",
+    "client = FhirApiClient(base_url=BASE_URL, fhir_version=FHIR_VERSION)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true,
+    "hidden": true
+   },
+   "source": [
+    "### Create a Patient Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "The FHIR base model was intended to include many common healthcare entities\n",
+    "like patients, biospecimens, etc. which cover 80% of healthcare data capture\n",
+    "uses cases.\n",
+    "\n",
+    "It is rare to come across a healthcare entity that isn't represented in\n",
+    "one of the existing FHIR resources. The remaining 20% of data capture use cases\n",
+    "that are not satisfied by the base model can easily be satisfied by extending\n",
+    "the base model.\n",
+    "\n",
+    "So we can start by looking at the base model's `Patient` `StructureDefinition` to\n",
+    "see if anything needs to be modified."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "The HL7 FHIR website has a nice visual representation of the `Patient`\n",
+    "`StructureDefinition`:\n",
+    "\n",
+    "[![Patient StructureDefinition](static/images/patient-sd-visual.png)](https://www.hl7.org/fhir/patient.html#resource)\n",
+    "📘Learn about the [symbol definitions in the view ^](http://hl7.org/fhir/StructureDefinition/contactpoint-area)\n",
+    "\n",
+    "We can already see that the `Patient` `StructureDefinition` has one attribute we want:\n",
+    "\n",
+    "- `gender`\n",
+    "\n",
+    "And many attributes that we'll want to **remove** because they represent PHI fields:\n",
+    "- `name`\n",
+    "- `telecom`\n",
+    "- `address`\n",
+    "- `birthDate`\n",
+    "- `contactPoint`\n",
+    "- `photo`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true,
+    "hidden": true
+   },
+   "source": [
+    "#### Start with a Template\n",
+    "We've seen that StructureDefinitions are complex looking structures. It is difficult to remember what attributes are required and the exact structure. One way to create StructureDefinitions is to use an application like Forge which hides that complexity since you're working with a UI. \n",
```

Comment:
```suggestion
    "We've seen that StructureDefinitions are complex. It is difficult to remember what attributes are required and the exact structure. Tools like Forge hide that complexity since you're working with a UI. \n",
```